### PR TITLE
Trigger node high load warnings sooner

### LINKED ISSUE #7552: [observability] update GitpodWorkspaceNodeHighNormalizedLoadAverage to trigger sooner
## Is your feature request related to a problem? Please describe

[Our observability is described here.](https://github.com/gitpod-io/gitpod/tree/main/operations/observability/mixins)

[We have existing alerts for node storage.](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/cross-teams/rules/components/nodes/alerts.libsonnet#L13)

[We have an existing warning](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet#L13), which indicates when nodes may be getting over whelmed, but it doesn't fire early enough. For example, this would be a signal that node resources (CPU and memory) are taxed, potentially degrading the experience for users in workspaces.

The current warning will not fire unless the condition has been met for [10 minutes](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet#L17), where the [normalized load is > 10](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet#L23).

## Describe the behaviour you'd like

The warning should fire when our nodes are under significant load. What we've seen, is that it's current params prevent firing even when load is significant.

A `for` of `5m` seems better than `10m`.
A `expression` > 5, seems better than 10.

In other words, if we cut what is currently there in half, and look at the data over the last 7 days, it would have triggered warnings (not errors) a few times. This would of given us a signal that a cluster was taxed, and the opportunity to investigate "the current state".

Inspect the load for the EU and US clusters for the last seven days. What do you think?

### How to test
We'll need to test using `loadgen`, I'm certain @aledbf knows how to do this, and would be willing to make a short Loom to help us use it. ;)

### Other
This is a warning, there is no runbook, which is okay, because it does not page.

Why do this? We encountered issues recently where load was high, but this alert did not fire.

### PR DESCRIPTION
**Pull request by Furisto**
## Description
Update GitpodWorkspaceNodeHighNormalizedLoadAverage to trigger sooner

## Related Issue(s)
Fixes #7552

## How to test
<strike>Use load testing with [loadgen](https://github.com/gitpod-io/gitpod/tree/main/dev/loadgen)?</strike>
Testing was done using:
```
apiVersion: v1
kind: Pod
metadata:
  name: test-alert-rules
spec:
  containers:
  - args:
    - --cpu
    - "256"
    - --timeout
    - 450s
    image: progrium/stress
    name: cpu-stress
  restartPolicy: Never
  nodeSelector:
    gitpod.io/workload_workspace: "true"
```

## Release Notes
```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **furisto** after the PR has been reviewed.

Associated issue: *#7552*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[operations/observability/mixins/workspace/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by kylos101** [comment]
@Furisto I think it would be good (if we tested this in a preview environment with [loadgen](https://github.com/gitpod-io/gitpod/tree/main/dev/loadgen), updating its readme accordingly.

Can you set this PR back to draft, and plan to do that related context, before flipping it back to ready?

@aledbf can you share any tips for using loadgen?

**Comment by Furisto** [comment]
The alert fired as expected
AlertUtilization
AlertFired

**Review by kylos101 — APPROVED** [review]
/LGTM

Thank you so much, @Furisto ! :pray:

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: f0ec0280f10d7fc7332e01fabaaab495ea52066d</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7710#pullrequestreview-859757557" title="LGTM">kylos101</a>*

Associated issue: *#7552*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/workspace/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/workspace/OWNERS)~~ [kylos101]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by kylos101** [comment]
/hold

@Furisto it looks like the build failed for the latest run, probably a fluke, can you rerun?

**Comment by Furisto** [comment]
There is actually already a newer run, but this was not detected somehow: https://werft.gitpod-dev.com/job/gitpod-build-furisto-7552.3

**Comment by kylos101** [comment]
/unhold 
build is passing :)

**Pull request merged**
